src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,72 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+public class Book {
+    private long id;
```

Comment:
The `id` field should be of type `Long` instead of `long` to match the task description and to allow for null values when the ID is not yet assigned.